aether-kernel/aether/kernel/api/views.py

```diff
@@ -53,7 +53,23 @@
 )
 
 
-class ProjectViewSet(MtViewSetMixin, viewsets.ModelViewSet):
+class FilteredMixin(object):
+    @action(detail=False, methods=['delete'], url_path='filtered-delete')
```

Comment:
Question, why not `bulk-delete`? :thinking:

Reply:
🤷‍♂  Bulk delete in my opinion feels like you have to pass the objects (ids) to be deleted.

Reply:
right